Gems/EMotionFX/Code/EMotionFX/CommandSystem/Source/RagdollCommands.cpp

```diff
@@ -143,7 +145,12 @@ namespace EMotionFX
         {
             AZStd::unordered_set<AZStd::string> parents;
 
-            Node* node = actor->GetSkeleton()->FindNodeByName(jointToAdd)->GetParentNode();
+            Node* node = actor->GetSkeleton()->FindNodeByName(jointToAdd);
```

Comment:
I can update if you want, but I think this is no riskier than the old code, because the old code was immediately doing GetParentNode which would blow up if the result of FindNodeByName was null.

Reply:
Ah okay in that case it's fine as is, maybe an Assert might be better then, or just leave as is :+1: